bin/crew

```diff
@@ -359,22 +360,6 @@ def files(pkg_name)
   EOT
 end
 
-def prop(silent = false)
-  props = []
-  pkg = Package.new
-  excluded_methods = %w[compatible binary source json_creatable autoload include const_defined class_variable_defined singleton_class method_defined public_method_defined private_method_defined protected_method_defined instance_variable_defined instance_of kind_of is_a frozen nil eql respond_to equal]
-  all_methods = pkg.class.methods.grep(/\?$/).to_s.gsub(/([?:,\[\]])/, '').split
-  all_methods.each do |method|
-    props.push(method) unless excluded_methods.include?(method)
-  end
```

Comment:
Add this to `lib/const.rb` to be used in 4 different places?

Reply:
How about this?
```
... calculated as above ...
CREW_BOOLEAN_PROPERTIES = props.join(', ')
```

Reply:
The whole point of this is to make `bin/crew` smaller and simpler by splitting it up into standalone (as much as possible) files-- doing this would work, but it would add `lib/package.rb` and `lib/const.rb` dependencies to `commands/prop.rb` and `commands/help.rb` and their tests, not to mention adding more complexity in what is supposed to be a simple list of static constants for easy reuse.

I think the downsides of having to manually update the property list are small (we aren't adding new properties very often, and when we do add them there's still manual work involved in doing so) compared to the penalties this incurs in code complexity and increased inter-dependency.

If you very strongly disagree, or if both you and @satmandu think this is the better path foward, that's fine (enough) by me.

Reply:
I'm open to a better property calculation algorithm.

Reply:
The problem isn't this particular algorithm, its using one at all, unfortunately.

Reply:
Not a problem, a solution.

Reply:
Resolved in #9519.